src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -70,15 +70,16 @@ CascadiaSettings::CascadiaSettings(const bool addDynamicProfiles)
 // Return Value:
 // - a non-ownership pointer to the profile matching the given guid, or nullptr
 //      if there is no match.
-const Profile* CascadiaSettings::FindProfile(GUID profileGuid) const noexcept
+const Profile CascadiaSettings::FindProfile(GUID profileGuid) const noexcept
 {
+    auto guid = winrt::guid(profileGuid);

```

Comment:
```suggestion
    const auto guid{ winrt::guid(profileGuid) };
```

const.

Reply:
no, actually

```suggestion
    const winrt::guid guid{ profileGuid };
```

`auto` is only valuable if you're not about to write the type on the righthand side anyway ;P